src/generator/AutoRest.Go/GoCodeGenerator.cs

```diff
@@ -60,7 +60,8 @@ public override void NormalizeClientModel(ServiceClient serviceClientModel)
         public override async Task Generate(ServiceClient serviceClient)
         {
             string packageName = GoCodeNamer.PackageNameFromNamespace(Settings.Namespace);
-            
+            string[] version = GoCodeNamer.SDKVersionFromPackageVersion(Settings.PackageVersion);
```

Comment:
Version fields should be considered optional so that we do not need it in testing.